src/cdk/datetime/datepicker-input.ts

```diff
@@ -197,33 +197,36 @@ export class CdkDatepickerInput<D> implements AfterContentInit, ControlValueAcce
   /** Implemented for datepicker CDK and locale subscriptions. */
   private readonly _subscriptions = new Subscription();
 
+  /** Prefix for form control validator properties. */
+  protected _prefix = 'cdk';
```

Comment:
Should this be a static property?

Reply:
I don't think so, since we are trying to override this property in the subclass: material datepicker input, and static properties don't work well with inheritance.